hooks/helpers/useOnChange.ts

```diff
@@ -0,0 +1,42 @@
+import { useErrorContext } from "hooks";
+import { ChangeEvent } from "react";
+
+export function useOnChange(
```

Comment:
this is really genericly named, but it seems like it does something really specific, might want to name it something more relevant

Reply:
agreed, will update